### PR TITLE
Update contributors list in CI

### PR DESCRIPTION
**Pull request by okuramasafumi**
Or at least it's an attempt to do so.
I hope someone who's familiar with GItHub Actions would review it!

### CODE REVIEW

File: .github/workflows/update-contributors.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+jobs:
+  update_contributors:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Update JSON
+        run: |
+          gh api -H "Accept: application/vnd.github+json" -H "X-GitHub-Api-Version: 2022-11-28" \
+                                                                                                   /repos/kaigionrails/conference-app/contributors > contributors.json
```
Comment (unasuke) [inline .github/workflows/update-contributors.yml:15]: Too many indentations?

Comment (okuramasafumi) [reply .github/workflows/update-contributors.yml]: Thx, fixed!

**Review by unasuke — APPROVED** [review]
looks yosasou

**Review by unasuke — CHANGES_REQUESTED** [review]
I found a bit concern

**Review by unasuke — APPROVED** [review]
yosasou again

**Comment by unasuke** [comment]
ref #86

**Pull request merged**